src/Transaction.ts

```diff
@@ -191,6 +191,16 @@ export class Transaction {
     public toBytes(): Uint8Array {
         return this._inner.serializeBinary();
     }
+
+    public toString(): string {
+        const tx = this.toProto().toObject();
+        const bodybytes = tx.bodybytes instanceof Uint8Array ?
+            Buffer.from(tx.bodybytes as Uint8Array) :
+            Buffer.from(tx.bodybytes as string, "base64");
+        tx.body = TransactionBody.deserializeBinary(bodybytes).toObject();
+
+        return tx.toString();
```

Comment:
Yeah, this isn't going to do what you expect: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString#Description

> If this method is not overridden in a custom object, toString() returns "[object type]", where type is the object type.

You probably want `JSON.stringify(tx)` instead.